CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.6.7
+
+- Fixed breaking change can't find file bug.
```

Comment:
could you pls make it more easy to understand ?

Reply:
yes. e.g. add some description  why cause this issue

Reply:
Updated. Add root cause analysis.